.gitignore

```diff
@@ -1,5 +1,8 @@
 .vscode
 .venv
+.devcontainer
+wrfout_d02_2019-03-05_12_00_00.nc
+tests/Lakes/input/hrrr.20190305.full
```

Comment:
Should this be expanded and for future files?
Think it also should move to the group below for consistency.

Reply:
Reorganized and cleaned out a lot of extra junk